acceptance/openstack/rds/v3/rds_test.go

```diff
@@ -35,12 +32,12 @@ func TestRdsLifecycle(t *testing.T) {
 
 	// Create RDSv3 instance
 	rds := createRDS(t, client, cc.RegionName)
-	defer deleteRDS(t, client, rds.Id)
+	t.Cleanup(func() { deleteRDS(t, client, rds.Id) })
 	th.AssertEquals(t, rds.Volume.Size, 100)
 
-	restart, err := instances.Restart(client, instances.RestartRdsInstanceOpts{}, rds.Id).Extract()
+	restart, err := instances.Restart(client, instances.RestartOpts{InstanceId: rds.Id})
 	th.AssertNoErr(t, err)
```

Comment:
rds_test.go:39: Failure in rds_test.go, line 39: unexpected error "Bad request with: [POST https://rds.eu-de.otc.t-systems.com/v3/a720688fb87f4575a4c000d818061eae/instances/9c497b94d2f84e839a3b18c5a01f9ee4in03/action], error message: {\"error_msg\":\"Parameter error.\",\"error_code\":\"DBS.280001\"}"